QgisModelBaker/tests/test_projectgen.py

```diff
@@ -3681,6 +3681,67 @@ def test_array_mapping_mssql(self):
 
         assert count == 1
 
+    def test_relation_editor_widget_bug637(self):
+
+        importer = iliimporter.Importer()
+        importer.tool = DbIliMode.ili2mssql
```

Comment:
Why mssql?

Reply:
Just because of copy/paste. The problem was happen independently of the provider. I can change it to postgis as it is our preferred provider.

Reply:
Was more out of curiosity. It does not matter in fact. But still, I prefer having postgres or gpkg for the test cases that not tests the database. Because when the test fails, it's easier to reproduce and follow back.

Reply:
I changed it to geopackage